spec/fixtures/site/_config.yml

```diff
@@ -1,7 +1,7 @@
 title: Your awesome title
 
 # Load the admin interface.
-plugins:
+gems:
```

Comment:
CAUTION: This reverts a recent change..

Reply:
You're right. But it gives me `Deprecation: The 'plugins' configuration option has been renamed to 'plugins_dir'. Please update your config file accordingly.` error. How do I fix this?

Reply:
That's probably because your `Gemfile.lock` is still using an older version of Jekyll. Running `bundle update` will remove the warning.

On an unrelated side-note, once `PR #322` gets merged, every other developer locked to an older version will thenceforth not see the warning..

Reply:
Yeah that was it. `bundle update jekyll` solved the issue. Thank you 👍